### ISSUE TITLE
no-extra-parens: false positive deprecation notics

### ISSUE THREAD

**Issue by coderaiser** [stylelint-stylistic/stylelint-stylistic#68]
### What minimal example or steps are needed to reproduce the bug?

run ESLint

### What minimal configuration is needed to reproduce the bug?

```json
  '@stylistic/no-extra-parens': ['error', 'all', {
      enforceForSequenceExpressions: false,
  }],
```

### How did you run Stylelint?

with ESLint

### Which Stylelint-related dependencies are you using?

stylystic plugin

### What did you expect to happen?

no deprecation notice

### What actually happened?

<img width="1162" height="26" alt="Image" src="https://github.com/user-attachments/assets/f24c214b-f05c-4d98-a02e-e8040ad9a1af" />

### Do you have a proposal to fix the bug?

the problem in deprecation notice
